packages/core/src/_common/url.js

```diff
@@ -118,21 +118,29 @@ const Url = (() => {
         return path.replace(/^\/br_.*?\//, '/');
     };
 
+    const getDerivComLink = (path = '') => {
+        const host = 'https://deriv.com';
+        const lang = getLanguage().toLowerCase();
+        const link_lang = lang === 'en' ? '' : `/${lang}`;
+
+        return `${host}${link_lang}${path}`;
```

Comment:
Please normalise the `path` that's passed in.

Reply:
path will be in `/path-to-page` format currently, what do you mean by normalize here?

Reply:
Thanks, addressed now.